client-web/src/exlibris-app/menu/exlibris-menu-item.html

```diff
@@ -0,0 +1,82 @@
+<link rel="import" href="../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../bower_components/paper-styles/color.html">
+
+<dom-module id="exlibris-menu-item">
+    <template>
+        <style>
+            :host {
+                display: block;
+            }
+
+            .item {
+                cursor: pointer;
+                margin: 4%;
```

Comment:
Margin in % is a bad idea. Are you sure about it? Usually it's better to use px. https://www.hongkiat.com/blog/calculate-css-percentage-margins/